ui/src/app/job-list/table/table.component.html

```diff
@@ -26,7 +10,7 @@
     <md-cell *mdCellDef="let j">
       <md-checkbox class = "checkbox" [checked]="isSelected(j)" (change)="toggleSelect(j)">
       </md-checkbox>
-      <a md-button class="job-details-button" [routerLink]="[j.id]">{{ j.name }}</a>
+      <a md-button class="job-details-button" [routerLink]="[j.id]" [queryParams]="getQueryParams()">{{ j.name }}</a>
```

Comment:
Why do we want to pass the query parameters to the job-details page?

Reply:
Ah okay I didn't realize the query builder would be on the details page, but that makes sense.